csrc/fusion_executor/executor_kernel_arg.h

```diff
@@ -135,4 +135,6 @@ std::vector<std::byte> getKernelArgument(
     Val* parameter,
     PrimDataType index_type);
 
+PolymorphicValue IValueToPolymorphicValue(const c10::IValue& val);
```

Comment:
should this util function be moved to another file?

Reply:
cc @zasdfgbnm -- I don't mind either way.